model/Check/System/PHPSessionTtlCheck.php

```diff
@@ -0,0 +1,116 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+namespace oat\taoSystemStatus\model\Check\System;
+
+use common_report_Report as Report;
+use oat\generis\persistence\PersistenceManager;
+use oat\taoSystemStatus\model\Check\AbstractCheck;
+use oat\oatbox\log\loggerawaretrait;
+use DateInterval;
+use DateTime;
+use Aws\ElastiCache\ElastiCacheClient;
+use oat\taoSystemStatus\model\SystemCheckException;
+use oat\awsTools\AwsClient;
+use oat\taoSystemStatus\model\Check\Traits\PieChartReportRenderer;
+
+/**
+ * Class PHPSessionTtlCheck
+ * @package oat\taoSystemStatus\model\Check\System
+ * @author Aleh Hutnikau, <hutnikau@1pt.com>
+ */
+class PHPSessionTtlCheck extends AbstractCheck
+{
+    use LoggerAwareTrait;
+
+
+    /**
+     * @param array $params
+     * @return Report
+     * @throws \Exception
+     */
+    public function __invoke($params = []): Report
+    {
+        $sessionMaxLifetime = (int) ini_get('session.gc_maxlifetime');
+        $cookieLifetime = (int) ini_get('session.cookie_lifetime');
```

Comment:
I wonder, can TAO work with cookies disabled?  If so what is the value of `session.cookie_lifetime` when cookies are disabled?

Reply:
For disabled (default one) is 0